src/app/progress/CrucibleRank.tsx

```diff
@@ -39,7 +39,9 @@ export function CrucibleRank({ progress, streak }: CrucibleRankProps) {
       </div>
       <div className={styles.factionInfo}>
         <div className={styles.factionLevel}>{progressionDef.displayProperties.name}</div>
-        <div className={styles.factionName}>{step.stepName}</div>
+        <div className={styles.factionName}>
+          {step.stepName} / Rank {progress.level + 1}
```

Comment:
`Rank` would need to be localised if this change is deemed useful

Reply:
Or put it on the line above 'Crucible Rank - #' then no i18n...

Reply:
I'm open to suggestions. Do you think there would be any benefit it showing the total rank too, similarly to how the XP is shown?:

`Crucible Rank (1 / 16)`

Please point me in the direction of your preferred solution and I'll make the changes

Reply:
I do not think there is any value to showing the denominator, as I believe the max rank is the same for all...
also I kind of lied about no i18n, we will need a transposition string `{{RankName}} - {{RankValue}}` so other languages can ensure the number goes where it should.

Reply:
I do like putting it in the RankName line, that's neat

Reply:
I've gone ahead and added the key for i18n in case you'd like to go ahead with this in addition to showing the rank on the Vendors page

By the looks of the Github workflow I'm guessing your translations are actually done elsewhere, but if it helps, this particular translation could be grabbed from the "Season Progress" page on bungie.net (e.g. https://www.bungie.net/7/Seasons/Progress)